conf/waives

```diff
@@ -0,0 +1,53 @@
+# TODO: unknown, but maybe expected as the rule description says that
+#   "The remediation for this rule uses firewall-cmd and nmcli tools.
+#    Therefore, it will only be executed if firewalld and NetworkManager
+#    services are running."
+# and since these are not running in an Anaconda %post environment,
+# maybe the remediation failing is expected .. ?
+# - see also https://github.com/ComplianceAsCode/content/issues/1639
+/hardening/anaconda/[^/]+/firewalld_sshd_port_enabled
+    rhel == 9 or rhel == 8
+
+# https://github.com/OpenSCAP/openscap/issues/1880
+# needs to be remediated more than once due to rule ordering issues
+/hardening/oscap/[^/]+/configure_bashrc_exec_tmux
+/hardening/oscap/[^/]+/no_tmux_in_shells
+/hardening/oscap/[^/]+/configure_usbguard_auditbackend
+    rhel == 9 or rhel == 8
+/hardening/oscap/[^/]+/configure_bashrc_tmux
+/hardening/oscap/stig(_gui)?/postfix_prevent_unrestricted_relay
+    rhel == 8
+
+# TODO: old test waives say this is also
+# https://github.com/OpenSCAP/openscap/issues/1880
+# but that seems to be incorrect as (unlike the others),
+# it happens in Anaconda remediation too
+/hardening/.+/accounts_password_pam_retry
+    rhel == 9 or rhel == 8
+
+# https://github.com/ComplianceAsCode/content/pull/10499
+# these are missing the required partitions, making the oscap Anaconda addon
+# error out during remediaton
+/hardening/anaconda/cis_(server|workstation)_l1
+    status == 'error' and note.startswith('RuntimeError: installation failed')
+
+# bz1828871, won't be fixed on rhel7
+# needs to be remediated more than once due to rule ordering issues
+/hardening/anaconda/[^/]+/postfix_network_listening_disabled
+    rhel == 7
+
+# bz1778661 abrt package
+/hardening/anaconda/e8/rpm_verify_(ownership|permissions)
+    rhel == 7
+
+# bz1825810 or maybe bz1929805
+# can be reproduced reliably both via anaconda and oscap CLI,
+# but apparently we don't really care about old releases
+/hardening/[^/]+/stig_gui/sysctl_net_ipv4_ip_forward
+    rhel == 7
+
+# TODO: something new? .. RHEL-8 on e8 and ism_o
```

Comment:
I'll leave it as TODO for now, investigation can probably be done elsewhere. It fails consistently, otherwise I'd have gotten errors (pass, but matched).